explore/src/main/scala/explore/tabs/ObsTabTiles.scala

```diff
@@ -336,6 +339,18 @@ object ObsTabTiles:
             )
           )
 
+        val timingWindows: View[List[TimingWindow]] =
```

Comment:
So timing windows go to their own tile? and removed from constraints?

Reply:
It already had its own tile, which was only shown in the constraints view. Now it is shown there and in the observation tabs.

If we create a new timing windows tab, we can show it there instead of the constraints tab.